darwin/DarwinMachine.h

```diff
@@ -9,6 +9,7 @@ in the source distribution for its full text.
 
 #include <mach/mach_host.h>
 #include <sys/sysctl.h>
+#include <IOKit/IOKitLib.h>
```

Comment:
Alphabetical, unless required differently (annotate if that's the case)

```suggestion
#include <IOKit/IOKitLib.h>
#include <mach/mach_host.h>
#include <sys/sysctl.h>
```